packages/derived-wallet-ethereum/src/index.ts

```diff
@@ -1,4 +1,5 @@
-export * from './EIP1193DerivedPublicKey';
-export * from './EIP1193DerivedSignature';
-export * from './EIP1193DerivedWallet';
-export * from './setupAutomaticDerivation';
+export * from "./EIP1193DerivedPublicKey";
+export * from "./EIP1193DerivedSignature";
+export * from "./EIP1193DerivedWallet";
+export * from "./setupAutomaticDerivation";
+export * from "./shared";
```

Comment:
did you need to use anything from "shared"?

Reply:
added by mistake, but I think we should still export everything, especially types

Reply:
Gotcha!
I had intentionally left it out, it's a utility function for internal use only
but I guess there's no harm done